docs/source/conf.py

```diff
@@ -495,6 +495,8 @@
     "barrier",
     "batch_isend_irecv",
     "broadcast",
+    "send_object_list",
```

Comment:
Hey!
Adding new entries in this file is not accepted anymore. Please make sure that your public APIs are properly documented and properly showing up on the doc website!

Reply:
Done and verified! Thanks for letting me know!

Reply:
ofc! I'll let @wconstab find the right person on the distributed team review the PR!